authors/apps/articles/urls.py

```diff
@@ -1,11 +1,18 @@
 from django.urls import path
 
 from authors.apps.articles import views
+from .models import Article, Reaction
 
 app_name = "articles"
 
 urlpatterns = [
     path('articles/', views.ListCreateArticle.as_view(), name='articles'),
     path('articles/<slug>/', views.RetrieveUpdateDeleteArticle.as_view(),
          name='article-details'),
+    path('articles/<slug>/like/', views.ReactionView.as_view(
+        model=Article, reaction_type=Reaction.LIKE),
```

Comment:
Unlike our other urls, you have models and reaction_type fields added to your urls.  What does this do? (especially the model part)

Reply:
reaction type is passed directly on the view when a request is sent to the endpoint. This defines the action type whether it is a  like or dislike.